multi-soil-moisture/dashboards/main/source.js

```diff
@@ -0,0 +1,30 @@
+function() {
+  return {
+    panels: {
+      latestMoisture: {
+        title: 'Latest Moisture',
+        type: 'ObservationCard',
+        data: {
+          topic: 'multi-soil-temperature-data-up-json',
+          field: 'moisture'
+        }
+      },
+      historicalMoisture: {
+        title: 'Moisture History',
+        type: 'ObservationTimeSeriesCard',
+        data: {
+          topic: 'multi-soil-temperature-data-up-json',
+          field: 'moisture'
+        }
+      }
+    },
+    layout: [
+      {
+        type: "row", entries: [
```

Comment:
Not sure how you feel about the importance of consistency with quotes (double / single)?

Reply:
good point